submissions/nauvian-perspective-bentham/nauvian-perspective-bentham.md

```diff
@@ -0,0 +1,46 @@
+
+It was around mid-2015 when he reached the monumental mid-point of his journey around the colossal lake, in his Railworld series. Coincidentally, at the same time, the 0.15 update dropped, bringing the Railworld preset. Bentham wondered if maybe the idea was from his Railworld series, so I asked Klonan (he created the preset) about it, who said "I want to keep some mystery in the world", so, we'll leave that thought at that :)*maybe remove this paragraph? seems like a nice thing to attribute to Bentham but then again, Klonan's answer...*
+
+Since 2015, Bentham has played some of my all-time favourite YouTube series with Factorio (obviously), [The Long Dark](https://www.youtube.com/watch?v=YPm-CEp4jYk&list=PLOx2-nvzE_ZLpGM2O2VIET2zT_Cu9WaSS), [Cortex Command](https://www.youtube.com/watch?v=fZZmMlNR_GU&list=PLOx2-nvzE_ZKub2IB8ksP7-SZ6j7FLkU8), [Astroneer](https://www.youtube.com/watch?v=nt2SCXqh-d0&list=PLOx2-nvzE_ZL_FIlc9y9s2JodKExG6_Uc), [Satisfactory](https://www.youtube.com/watch?v=lVMNn4VDOVU&list=PLOx2-nvzE_ZJfc-WhE42WNaOusForSHeA) and [Dyson Sphere Program](https://www.youtube.com/watch?v=ElFjOPyrlts&list=PLOx2-nvzE_ZLGjohS04LpsagHgRK31iON). Like Zisteau (covered in our [last Nauvian Perspective article](https://alt-f4.blog/ALTF4-30/)), Bentham played Satisfactory extensively, although not in the same playstyle.
+
+Now, busy with real life and being an admin of the tabletop games, roleplaying and D&D channel [Roll4It](https://www.youtube.com/channel/UCifkjVaOOT5VxraQpYUwL6A), Bentham's personal channel has gone a bit spotty uploads-wise. Still, he can boast 1.5k hours of Factorio on Steam, and at least double that more from before the game release on Steam. He's recently had some continuation of his [Factslowrio](https://www.youtube.com/watch?v=X388d9waOEg&list=PLOx2-nvzE_ZJGpYLwW3vJSAKqVhECOsz-) April Fools series that isn't really a joke anymore... *Not really sure what to say here?*
+
+## Xisumavoid
+Whilst researching Bentham's first Factorio series, I came across a lot of comments from [Xisumavoid](https://www.youtube.com/c/XisumavoidMC/featured). I thought this was interesting because although I watched his [late 2014 Factorio series](https://www.youtube.com/watch?v=lKdAhv_LkLM&list=PL7VmhWGNRxKjVgYaRnIMZz31i1F9rNUBL&index=1), I didn't realise he had played the game all the way back in its very early days [in 2013](https://www.youtube.com/watch?v=ga-y3rXzVeo&list=PL7VmhWGNRxKjKohJTh9utStd3XdLN0q6h). Xisuma told me that he first heard of Factorio over 8 years ago! He couldn't remember where he stumbled upon it however he remembered being drawn to its visual style, which is a quality that he enjoys from games. I see a lot of similarities in Xisuma and Bentham - they are both down-to-earth and have similar humour (plus are both English!).
```

Comment:
I ended up on integrating those details, aside from the latter part, into the Xisuma introductory paragraph. I omitted the latter part because by that point I think it would be too much background information.